src/prr.rs

```diff
@@ -678,4 +686,80 @@
         let (prr, _dir) = config(gconfig, None);
         assert!(prr.workdir().is_err());
     }
+
+    fn copy_dir_all(src: impl AsRef<Path>, dst: impl AsRef<Path>) -> io::Result<()> {
```

Comment:
Nit: might be a bit easier to read overall to have this helper be infallible. And make
the body of the function panic at the right places. Cuz it's a test -- we will never try
to recover from errors.